R/sample_and_adjust.R

```diff
@@ -63,7 +66,8 @@ sample_and_adjust_by_dose <- function(
   ruv_i <- generate_ruv(
     tdm_sample_time = tdm_times,
     prop = sim_ruv$prop,
-    add = sim_ruv$add
+    add = sim_ruv$add,
+    ltbs=ltbs
```

Comment:
This way the user doesn't have to think about if the model should be ltbs

```suggestion
    ltbs = isTRUE(attr(sim_model, "ltbs"))
```

`isTRUE` is a handy function that returns FALSE is the field is missing/NULL/FALSE.

Reply:
Awesome! Might be silly/obvious but attr() is also nice. I had this thought but wasn't sure how to check/get the field out of the model with its unique object status.